src/parser/core/modules/EventEmitter.ts

```diff
@@ -257,24 +278,26 @@ class EventEmitter extends Module {
       return this.triggerEvent(fabricatedEvent);
     }
   }
-  _validateEvent(event) {
+  _validateEvent(event: any) {
     if (!event.type) {
       console.log(event);
       throw new Error('Events should have a type. No type received. See the console for the event.');
     }
   }
-  _handleError(err, module) {
+  _handleError(err: Error, module: Module) {
     if (process.env.NODE_ENV !== 'production') {
       throw err;
     }
     const name = module.key;
     console.error('Disabling', name, 'and child dependencies because an error occured:', err);
     // Disable this module and all active modules that have this as a dependency
-    this.owner.deepDisable(module, MODULE_ERROR.EVENTS, err);
+    this.owner.deepDisable(module, ModuleError.EVENTS, err);
+    // @ts-ignore Sentry does not exist on window, need Sentry types
```

Comment:
I got you fam

Reply:
**sentry.d.ts**
```ts
import * as SentryType from '@sentry/browser'

declare global {
  // May be undefined when adblockers block Sentry (for privacy concerns)
  const Sentry: typeof SentryType | undefined
}
```

Reply:
this will define Sentry but not window.Sentry, will it work if I change the code to `Sentry && Sentry.withScope` ? Or do I need to define Sentry onto `window`

Reply:
it should work without window, otherwise I have other projects I need to fix

Reply:
Cool beans!

Reply:
![image](https://user-images.githubusercontent.com/4565223/87827232-c121bb00-c87a-11ea-80f4-5fd0fbb79ad1.png)

bigoof

Reply:
you are right
I had it wrong
do yours